tests/publiccloud/storage_perf.pm

```diff
@@ -99,19 +100,20 @@ sub analyze_previous_series {
         my $previous_records_mean = get_mean_from_db($args, " limit 5 offset 5");
         # we do analysis only if we get back some non-zero values
         if (defined($previous_records_mean) && defined($last_records_mean) && $previous_records_mean != 0 && $last_records_mean != 0) {
-            record_info('ANALYZE', "Checking " . $current_load_type);
-            # this formula detect if mean values differs more than 10%
-            if ((abs($previous_records_mean - $last_records_mean) / $previous_records_mean) * 100 > 10) {
-                die sprintf("Anomaly occurred in http://openqa-perf.qa.suse.de/d/hoRc37HWz/storage-performance?orgId=1&var-os_flavor=%s&var-os_version=%s", $args->{os_flavor}, $args->{os_flavor});
+            my $diff_percents = (abs($previous_records_mean - $last_records_mean) / $previous_records_mean) * 100;
+            record_info('ANALYZE', sprintf("Checking %s:\nFormula: (abs(\$previous_records_mean - \$last_records_mean) / \$previous_records_mean) * 100 = (abs(%s - %s) / %s) * 100 = %s%", $current_load_type, $previous_records_mean, $last_records_mean, $previous_records_mean, $diff_percents));
+            # This detects if mean values differs more than 10%
+            if ($diff_percents > 10) {
+                record_soft_failure(sprintf("Anomaly occurred in http://openqa-perf.qa.suse.de/d/hoRc37HWz/storage-performance?orgId=1&var-os_flavor=%s&var-os_version=%s", $args->{os_flavor}, $args->{os_flavor}));
```

Comment:
One important information is missing in all of those output: What value is causing the deviation? Is it the load? CPU time? Disk storage? Number of bananas on the system?

Also: Can we please not call this anomaly but a deviation please? :slightly_smiling_face:

Reply:
> One important information is missing in all of those output: What value is causing the deviation? Is it the load? CPU time? Disk storage? Number of bananas on the system?

Well I would expect from a person looking into **storage_perf* test to be aware that he/she reviewing results of storage performance . As Pavel correctly mention just one record_info before this one we giving some hints . I don't think it is necessary to repeat that again . 

> Also: Can we please not call this anomaly but a deviation please? 

:+1:  yeah your deviation is certainly better than anomaly .